src/components/dialog/dialog.spec.js

```diff
@@ -243,7 +244,7 @@ describe('$mdDialog', function() {
           ok: 'Next',
           cancel: 'Back',
           title: 'Which Way ',
-          content: '<div class="mine">Choose</div>'
+          template: '<div class="mine">Choose</div>'
```

Comment:
@jelbourn - this checks the `template` option... but what about the `htmlContent` option